### PR TITLE
Cli patch

### PR DESCRIPTION
**Pull request by TJKoury**
During the talk this morning tried to run 'vue list' but ran up against API rate limit (as we're all hitting the server).  This helps handle the error gracefully.

### CODE REVIEW

File: bin/vue-list [hunk 1]
```diff
@@ -18,18 +19,32 @@ process.on('exit', function () {
...
-  })
+  if (err) {
+    logger.fatal(err)
+  } else {
+    var _body = JSON.parse(body)
+    if (Array.isArray(_body) &&
+        _body.length) {
+      console.log('  Available official templates: \n')
+      _body.forEach(function (repo) {
+        console.log('%s %s - %s',
```
Comment (posva) [inline bin/vue-list]: Aren't you missing some whitespace from lines 31 to 32?

File: bin/vue-list [hunk 2]
```diff
@@ -18,18 +19,32 @@ process.on('exit', function () {
...
-  JSON.parse(body).forEach(function (repo) {
-    console.log(
-      '  ' + chalk.yellow('★') +
-      '  ' + chalk.blue(repo.name) +
-      ' - ' + repo.description)
-  })
+  if (err) {
+    logger.fatal(err)
+  } else {
+    var _body = JSON.parse(body)
```
Comment (posva) [inline bin/vue-list]: I personally would override the `body`variable since it isn't used anymore

File: bin/vue-list [hunk 3]
```diff
@@ -18,18 +19,32 @@ process.on('exit', function () {
...
+          chalk.yellow('★'),
+          chalk.blue(repo.name),
+          repo.description)
+      })
+    } else if (!_body.length) {
+      console.log('%s %s',
+        chalk.yellow('No Templates found:'),
+        templateUrl)
+    } else if (_body.hasOwnProperty('message')) {
+      console.log(chalk.yellow(_body.message))
```
Comment (posva) [inline bin/vue-list]: Is this when the API has an issue? If so, it may be valuable to say it

**Comment by TJKoury** [comment]
@posva Made a few changes based on your comments.  Let me know if there's anything else.

**Comment by posva** [comment]
@TJKoury Are you looking into tests?

**Comment by TJKoury** [comment]
Huh, strange it was passing before.  I'll check it out.

**Comment by zigomir** [comment]
Already fixed with https://github.com/vuejs/vue-cli/pull/131

**Comment by TJKoury** [comment]
Looks good, thanks!

**Pull request closed without merging**